config/index.js

```diff
@@ -244,6 +244,11 @@ var conf = convict({
       format: String,
       default: 'https://support.mozilla.org/kb/password-manager-remember-delete-change-and-import#w_viewing-and-deleting-passwords'
     },
+    accountSettingsUrl: {
+      doc: 'url to Firefox account settings page',
+      format: String,
+      default: 'https://accounts.firefox.com/settings'
```

Comment:
You probably do not need this config value here, you can derive it from `contentServer.url`. See bottom of this file for an example where those values are set.